controllers/discovery/searchUser.js

```diff
@@ -11,77 +11,92 @@ const {sequelize} = require('../../databases/models');
 const SearchUser = async (req, res) => {
   const {q, limit = 50} = req.query;
   const {userId} = req;
+
   if (q.length < 2)
     return res.status(200).json({
       success: true,
       message: 'Your search characters is too few, please input 3 or more characters for search'
     });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The check for the length of the search query is less than 2. This could be problematic if a user wants to search for users with one-character usernames. Consider reducing this limit or removing it entirely if your system allows one-character usernames.
```diff
-    if (q.length < 2)
+    if (q.length < 1)
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->